### ISSUE TITLE
Studio does not honor JDBC url database parameter

### ISSUE THREAD

**Issue by deceiver08** [TIBCOSoftware/jasperreports#128]
When I try to create a data adapter to a Azure SQL database, the connection always goes to the master database rather than the database I specify in the jdbc url using the databaseName parameter (I also tried just "database" too for the parameter and it didn't work either). The user I am using does have rights to the database called out. I can't move forward.

**Comment by teodord** [comment]
What is the JDBC driver (name and version) that you are using?

**Comment by deceiver08** [comment]
![image](https://user-images.githubusercontent.com/63079498/80510134-28d90180-892f-11ea-986f-7a7baec859be.png)

This is the master database. Not the db I specify in the connection string.
![image](https://user-images.githubusercontent.com/63079498/80509434-435eab00-892e-11ea-90f4-6a003e235586.png)

**Comment by teodord** [comment]
I am not familiar with how this driver works, but just by reading its documentation, seems that the URL parameter is called "databaseName":
https://docs.microsoft.com/en-us/sql/connect/jdbc/using-the-jdbc-driver?view=sql-server-ver15

JasperReports is not doing anything with the URL and it passes it to the driver, which is responsible to make use of it, the way it sees fit.

You should try use the driver with some other Java applications to see if it works there. I suggest a Java SQL tool such as Squirrel.
http://squirrel-sql.sourceforge.net/

So let's put aside JRL and JSS for a moment and see if the driver actually works as advertised.

**Comment by deceiver08** [comment]
Hi @teodord. Yeah I caught that. I use the databaseName parameter. Sorry for the confusion.

I did as you suggested and Squirrel connects just fine using the driver. As you can see, in Squirrel  both databases are presented, whereas in Jasper Studio its just master. I am literally using the exact same connection strings between the two software?
![image](https://user-images.githubusercontent.com/63079498/80543426-b5051c00-8963-11ea-9547-e428ff8802f3.png)

**Comment by deceiver08** [comment]
If you are aware of a different java driver that will work with an Azure Sql database in Studio, let me know. At the moment, this java driver is the latest and greatest from Microsoft directly.

**Comment by teodord** [comment]
Looks like this Squirrel test did not help us much after all. From what I can see, the fact that you provided the databaseName to the Squirrel URL did not make it see only that database. Squirrel will see all databases anyway. Probably even if you don't specify a databaseName at all, right?

**Comment by deceiver08** [comment]
You are right. I stripped out the databaseName parameter and Squirrel still presented me with both databases. I tried the same thing in Studio and it still shows just master. So if they are both using the same driver, is something in Studio hardcoded to filter or only present one db regardless of driver?
![image](https://user-images.githubusercontent.com/63079498/80620119-6eaecc00-89fa-11ea-86ba-6f045aed33ca.png)
